### DISCUSSION TITLE
Quiz the meaning from the Kanji

### DISCUSSION THREAD

**Discussion by francoisthire** [fabd/kanji-koohii#238] [1. Feedback & Feature Requests]
From the moment, the only type of question I have is to guess the kanji from the RTK keyword associated. Having the opposite would be interesting too, no?

**Comment by fabd** [comment]
Hi @francoisthire 

Please remember the keyword is not exactly the meaning, but I know what you mean, the english keyword *often* does relate to the meaning, or one of the meanings of the character when it is used in words.

More to the point, there is a way to review **kanji to keyword** , however it is not available in SRS modes.

You can check the  ☑ **Kanji to Keyword (reverse mode)** option in the [Kanji Review](https://kanji.koohii.com/review/custom) page.

**Reply by francoisthire** [reply]
> Please remember the keyword is not exactly the meaning, but I know what you mean, the english keyword often does relate to the meaning, or one of the meanings of the character when it is used in words.

Thanks for the reminder, I indeed should have said the keyword.

> More to the point, there is a way to review kanji to keyword , however it is not available in SRS modes.

Is there any reason?

**Reply by fabd** [reply]
Up until now I guess I've been stubborn in following Heisig's advice : which is to only review from keyword to the kanji, and let the other direction of recall happen as you work with real Japanese text.

However I've changed my stance lately in general I feel more like just implementing things that people need.. but I still need to be motivated enough.

What doesn't make sense imho is to have the SRS store review status separately for **kanji > keyword** and **keyword > kanji**. That is twice as much storage, two SRS charts, etc. *and* probably more reviews than most people want to handle. It's time consuming enough for most users to review 2000+ kanji after completing RTK.

I could see myself adding a "reverse mode" option of some sort in the SRS Settings. But if I do that that means you chose to  always SRS review from kanji > keyword.

I guess even if I am not convinced of the benefits of doing that, if this gets more votes I might just eventually implement the option.

**Reply by francoisthire** [reply]
>  Heisig's advice : which is to only review from keyword to the kanji, and let the other direction of recall happen as you work with real Japanese text.

I missed that advice.

I think for all the arguments you pointed above, the current situation is ok. I don't really think it needs to be in SRS. 

Something doable maybe is that the `kanji to keyword` reviewing mode has an impact on the SRS score? (optionally I mean)

Likely, if you don't remember the `kanji to keyword` sense, it means you should review the kanji more often? It is only an intuition, maybe I am completely wrong.
